website/blog/2023-11-15-run-webassembly-wasm-workloads-windows-and-macos.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Use WebAssembly on macOS and Windows
+description: Spinning a OCI container image containing a WebAssembly/WASM workload on macOS or Windows should be as simple as running any other OCI image.
+slug: wasm-workloads-on-macos-and-windows-with-podman
+authors: [benoitf]
+tags: [podman-desktop, wasm, wasi, WebAssembly]
+hide_table_of_contents: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+How to quickly run WebAssembly/wasm binaries ?
```

Comment:
Intros should be conversational, pull you in, and get you (at least a little) excited about the topic, but compared to the rest of the blog this is reading rather flat to me at the moment. Should not include the phrase 'use case'. :)

It needs something like 'You may heard a lot of excitement recently about WASM and WASI. But what are they, and what can I do with them? This blog will give some background and show how you can use Podman to run these applications on macOS and Windows today.'

Feel free to do something else entirely, just needs to be punchier and a little marketing.

Reply:
this is a good remark 👍

Reply:
I've reworked intro